concepts/array-destructuring/about.md

```diff
@@ -84,15 +86,14 @@ fourth;
 
 The array destructuring assignment can provide _default values_ in case there is none in the source array.
 
-```
-const [first, second, third="none selected", fourth] = pickAtLeastOne
+```javascript
+const [first, second, third = 'none selected', fourth] = pickAtLeastOne;
 
-third
+third;
 // => 'none selected'
 
-fourth
+fourth;
 // => undefined
 ```
 
-See: `rest-elements`
-See: `spread-elements`
+See: `rest-and-spread`
```

Comment:
Could you replace this line with the respective widget: https://exercism.lol/docs/building/markdown/widgets
I think they are working now.

Reply:
Yup: [`d5d6ed9` (#1274)](https://github.com/exercism/javascript/pull/1274/commits/d5d6ed99235cc02b9b6c8615b9b4acee0f43e901)

I saw them working on the Julia track but then something changed and Julia's syllabus is no longer available.